controller_manager:
  ros__parameters:
    update_rate: 30

    forward_position_controller:
      type: forward_command_controller/ForwardCommandController

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
    
    velocity_controller:
      type: velocity_controllers/JointGroupVelocityController
    

forward_position_controller:
  ros__parameters:
    joints:
      - front_left_steering_joint
      - front_right_steering_joint
    interface_name: position

velocity_controller:
  ros__parameters:
    joints:
      - front_left_wheel_joint
      - front_right_wheel_joint